app/javascript/components/patient/Patient.js

```diff
@@ -72,6 +74,78 @@ class Patient extends React.Component {
     }
   }
 
+  renderContactInfo() {
+    let contactInfoPatient = this.state.displayHohContactInfo ? this.props.hoh : this.props.details;
+    let switchContactInfoButton;
+    if (this.state.isMinor && this.props.hoh) {
+      switchContactInfoButton = (
+        <Button
+          id="switch-contact-info"
+          variant="link"
+          className="p-0"
+          aria-label="switch displayed contact info"
+          onClick={() => this.setState({ displayHohContactInfo: !this.state.displayHohContactInfo })}>
+          <span>View Contact Info for {this.state.displayHohContactInfo ? 'Monitoree' : 'Reporter'}</span>
+        </Button>
+      );
+    }
+    return (
+      <Col id="contact-information" lg={10} className="col-xxl-12">
+        <div className="section-header">
+          <h4 className="section-title">Contact Information</h4>
+          {this.state.displayHohContactInfo ? '' : this.renderEditLink('Contact Information', 2)}
+        </div>
+        <div className="item-group">
+          <div>
+            <span style={{ color: 'red' }}>{this.state.isMinor ? 'Monitoree is a minor.' : ''}</span>
+          </div>
+          <div>{switchContactInfoButton}</div>
+          <div>
```

Comment:
Why the empty span here? just wrap the whole div in `{this.state.displayHohContactInfo && ( )}`